install/host_preparation.adoc

```diff
@@ -283,6 +283,11 @@ install the following package:
 # yum install openshift-ansible
 ----
 +
+[NOTE]
+====
+If you face an error while running `yum install openshift-ansible` command, you might need an additional repository to proceed with the installation. For more information, see https://access.redhat.com/solutions/3949501.
+====
++
```

Comment:
Per the KCS link, the issue is already fixed in the latest release, I do not think using an additional repository is a good idea, but only link customers to reference this KCS link.

Reply:
Addressed